kernel/src/analysis/javascript.rs

```diff
@@ -17,122 +17,7 @@ const JAVASCRIPT_EXECUTION_TIMEOUT_MS: u64 = 5000;
-        this.category = "SAFETY";
-        this.fixes = [];
-        this.addFix = function(fix) {
-            this.fixes.push(fix);
-            return this;
-        }
-    }
-
-    function StellaConsole(startLine, startCol, endLine, endCol, message, severity, category) {
-        this.lines = [];
-        this.log = function(message) {
-            this.lines.push(message);
-        }
-    }
-
-    console = new StellaConsole();
-
-    function StellaFix(message, edits) {
-        this.description = message;
-        this.edits = edits;
-    }
-
-    function StellaEdit(start, end, editType, content) {
-        this.start = start;
-        this.end = end;
-        this.editType = editType;
-        this.content = content;
-    }
-
-    function buildError(startLine, startCol, endLine, endCol, message, severity, category) {
-        return new StellaError(startLine, startCol, endLine, endCol, message, severity, category);
-    }
-
-    function buildFix(message, list) {
-        return new StellaFix(message, list);
-    }
-
-    function buildEditUpdate(startLine, startCol, endLine, endCol, content) {
-        return new buildEdit(startLine, startCol, endLine, endCol, "UPDATE", content);
-    }
-
-    function buildEditRemove(startLine, startCol, endLine, endCol) {
-        return new buildEdit(startLine, startCol, null, null, "REMOVE");
-    }
-
-
-    function buildEditAdd(startLine, startCol, content) {
-        return new buildEdit(startLine, startCol, null, null, "ADD", content);
-    }
-
-
-    function buildEdit(startLine, startCol, endLine, endCol, editType, content) {
-        const start = {
-            line: startLine,
-            col: startCol,
-        };
-
-        let end = {
-            line: endLine,
-            col: endCol,
-        };
-
-        if (!endLine || !endCol) {
-           end = null;
-        }
-        return new StellaEdit(start, end, editType.toUpperCase(), content);
-    }
-
-    function addError(error) {
-        stellaAllErrors.push(error);
-    }
-
-    // helper function getCode
-    function getCode(start, end, code) {
-        const lines = code.split("\n");
-        const startLine = start.line - 1;
-        const startCol = start.col - 1;
-        const endLine = end.line - 1;
-        const endCol = end.col - 1;
-
-        var startChar = 0;
-        for (var i = 0 ; i < startLine ; i++) {
-            startChar = startChar + lines[i].length + 1;
-        }
-        startChar = startChar + startCol;
-
-        var endChar = 0;
-        for (var i = 0 ; i < endLine ; i++) {
-            endChar = endChar + lines[i].length + 1;
-        }
-        endChar = endChar + endCol;
-
-        return code.substring(startChar, endChar);
-    };
-
-    // helper function getCodeForNode
-    function getCodeForNode(node, code) {
-        return getCode(node.start, node.end, code);
-    }
-        "#,
-        );
+        let code: FastString = FastString::from_static(include_str!("./js/stella.js"));
```

Comment:
Does this work also for Windows? Are the paths with `/` supported on windows or will the build fail?

Reply:
Can you rebase this on the current `main`? Test for windows will then execute and we can test that the change works for Windows.

Reply:
Rebased

Reply:
Good point about the path. I'm so used to work with Unix systems these days that I haven't thought of that. Will test it.

Reply:
According to [this](https://users.rust-lang.org/t/why-does-include-str-act-differently-on-different-oss/43314) it should work across OSs but I'll try to double check that anyways.

Reply:
Double-checked! It works in Windows!

Reply:
Nice! Thank you for checking!